iron-doc-nav.html

```diff
@@ -110,14 +110,24 @@
 
         _descriptorChanged(descriptor) {
           descriptor = descriptor || {};
-          this._sections = [
+
+          var sections = [
             _section('Namespaces', '/namespaces/', descriptor.namespaces),
             _section('Elements', '/elements/', descriptor.elements),
             _section('Behaviors', '/behaviors/',
                 ((descriptor.metadata || {}).polymer || {}).behaviors),
             _section('Mixins', '/mixins/', descriptor.mixins),
             _section('Classes', '/classes/', descriptor.classes),
           ];
+
+          var pruned = [];
+          for (var i = 0; i < sections.length; i++) {
+            if (sections[i].items.length) {
+              pruned.push(sections[i]);
+            }
+          }
+
+          this._sections = pruned;
```

Comment:
Another technique I've used is to use data binding.

```
<section hidden$="[[!list.length]]">
...
```

Would that be more suitable than this pruning approach?